pom.xml

```diff
@@ -45,6 +45,9 @@
     <properties>
         <java.version>1.8</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <sonar.organization>bitwarden</sonar.organization>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+        <sonar.projectKey>bitwarden_passwordless-java</sonar.projectKey>
```

Comment:
❓ With these above in the Sonar invocation can they be removed here?

Reply:
These are required in the pom.xml by SonarCloud to support maven projects.